tests/testthat/test_preprocess_count_data.R

```diff
@@ -0,0 +1,184 @@
+# Test data setup
+hosp_data <- tibble::tibble(
+  date = lubridate::ymd(c("2023-11-01", "2023-11-02")),
+  daily_admits = c(10, 20),
+  state_pop = c(1e6, 1e6)
+)
+
+# Test that function returns a dataframe with correct columns
+test_that("Function returns dataframe with correct columns", {
+  processed <- preprocess_count_data(hosp_data,
+    count_col_name = "daily_admits",
+    pop_size_col_name = "state_pop"
+  )
+
+  expected_cols <- c("date", "count", "total_pop")
+
+  checkmate::expect_names(names(processed),
+    must.include = expected_cols
+  )
+})
+
+# Test that count column is renamed correctly
+test_that("Count column is renamed correctly", {
+  processed <- preprocess_count_data(hosp_data,
+    count_col_name = "daily_admits",
+    pop_size_col_name = "state_pop"
+  )
+
+  checkmate::expect_names(names(processed),
+    must.include = "count",
+    disjunct.from = "daily_admits"
+  )
+})
+
+# Test that population size column is renamed correctly
+test_that("Population size column is renamed correctly", {
+  processed <- preprocess_count_data(hosp_data,
+    count_col_name = "daily_admits",
+    pop_size_col_name = "state_pop"
+  )
+
+  checkmate::expect_names(names(processed),
+    must.include = "total_pop",
+    disjunct.from = "state_pop"
+  )
+})
+
+# Test data setup
+hosp_data <- tibble::tibble(
+  date = lubridate::ymd(c("2023-11-01", "2023-11-02")),
+  daily_admits = c(10, 20),
+  state_pop = c(1e6, 1e6)
+)
+
+# Test that function returns a dataframe with correct columns
+test_that("Function returns dataframe with correct columns", {
+  processed <- preprocess_count_data(hosp_data,
+    count_col_name = "daily_admits",
+    pop_size_col_name = "state_pop"
+  )
+
+  expected_cols <- c("date", "count", "total_pop")
+  expect_true(checkmate::check_names(names(processed),
+    must.include = expected_cols
+  ))
+})
+
+# Test that count column is renamed correctly
+test_that("Count column is renamed correctly", {
+  processed <- preprocess_count_data(hosp_data,
+    count_col_name = "daily_admits",
+    pop_size_col_name = "state_pop"
+  )
+
+  expect_false("daily_admits" %in% names(processed))
+  expect_true(checkmate::check_names(names(processed), must.include = "count"))
```

Comment:
```suggestion
checkmate::expect_names(names(processed), must.include = "count", disjunct.from = "daily_admits")
```

Reply:
Oh my bad I initially wrapped these bc I thought they worked like `check` and just returned TRUE or a string error message. But this function luckily does error. I will note that CI views this as an empty test because it doesn't use test that infrastructure I think

Reply:
Interesting. `checkmate`'s `expect_` functions are designed to work with `testthat`. I wonder if this would still be viewed as an empty test if you ran `library(checkmate)` after `library(testthat)` at the top of your test.

Another workaround would just be to add in a `expect_true(TRUE)`, but that's a bit of an unpleasant workaround.

https://mllg.github.io/checkmate/articles/checkmate.html#checkmate-as-testthat-extension

Reply:
Ok now I am not seeing it anymore so maybe it was another issue